pages.es/linux/systemd-tmpfiles.md

```diff
@@ -0,0 +1,25 @@
+# systemd-tmpfiles
+
+> Crea, elimina y limpia archivos y directorios volátiles y temporales.
+> Este comando es invocado automáticamente en el arranque por los servicios systemd, en tanto ejecutarlo manualmente no suele ser necesario.
+> Más información: <https://www.freedesktop.org/software/systemd/man/systemd-tmpfiles.html>.
+
+- Crea los archivos y directorios especificados en la configuración:
+
+`systemd-tmpfiles --create`
+
+- Limpia archivos y directorios con los parámetros de edad configurados:
+
+`systemd-tmpfiles --clean`
+
+- Elimina archivos y directorios según lo especificado en la configuración:
+
+`systemd-tmpfiles --remove`
+
+- Aplica operaciones para configuraciones específicas de usuario:
+
+`systemd-tmpfiles --create --user`
+
+- Ejecuta las líneas marcadas para el arranque anticipado:
```

Comment:
```suggestion
- Ejecuta las líneas marcadas para el inicio del arranque:
```

I am not 100% percent sure of  the translation from `early` to `inicio`, what do you think @kant?

Reply:
Maybe "Ejecuta las líneas marcadas para el arranque inicial"

Reply:
But by early I think the original author meant to say at the first moments of the boot process not the first boot.

I will search and ask the author.

Reply:
Hello @quantumflo! We are translating your page to Spanish and were wondering what you meant by ``...early boot"?

Reply:
This is what I found in the man page locally:

```roff
--create
If this option is passed, all files and directories marked with f,
F, w, d, D, v, p, L, c, b, m in the configuration files are created
or written to. Files and directories marked with z, Z, t, T, a, and
A have their ownership, access mode and security labels set.

...

--boot
Also execute lines with an exclamation mark. Lines that are not
safe to be executed on a running system may be marked in this way.
systemd-tmpfiles is executed in early boot with --boot specified
and will execute those lines. When invoked again later, it should
be called without --boot.
```

IG this command marks the file to be executed in the early/first boot (the first process) after the system starts.

Reply:
I think `al inicio` fits for now given the information from the man page, is it okay if I apply the suggestion?

Reply:
ok!